webpack.config.js

```diff
@@ -11,6 +12,10 @@ const baseConfig = {
   module: {
     loaders: [
       {
+        test: /\.css$/,
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'),
```

Comment:
Oh i see - move CSS compilation to `devAppConfig`? That would be for dev - how do we enable CSS compilation when on lambda?